contrib/agent/src/main/java/io/opencensus/contrib/agent/deps/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2017, Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.agent.deps;
```

Comment:
Currently, the only dependency repackaged by the build script to this location is Bytebuddy (https://github.com/census-instrumentation/opencensus-java/blob/67683b6c8c3a6b675de8a96dbb3f00ca420c1758/contrib/agent/build.gradle#L47-L49). I intend to add more deps over time, such as Guava. This will increase the size of the resulting agent JAR file, but Proguard will help to shrink it a lot.

Moving this to io.opencensus.contrib.agent.internal.deps SGTM. Going to update the PR accordingly.